resourcecontrollerv2/resource_controller_v2.go

```diff
@@ -2266,6 +2266,12 @@ func (options *CreateResourceKeyOptions) SetHeaders(param map[string]string) *Cr
 
 // Credentials : The credentials for a resource.
 type Credentials struct {
+	// If present, the user doesn't have the correct access to view the credentials and the details are redacted.  The
+	// string value identifies the level of access that's required to view the credential. For additional information, see
+	// [viewing a
+	// credential](https://cloud.ibm.com/docs/account?topic=account-service_credentials&interface=ui#viewing-credentials-ui).
+	Redacted *string `json:"REDACTED,omitempty"`
```

Comment:
So where is this coming from? Was it added by hand? I don't see it in the API definition

Reply:
We have updated the api definition, the PR is merged in test, It needs to be merged in production - https://github.ibm.com/cloud-api-docs/resource-controller/pull/248